common/v2/features/SendAssets/components/TransactionReceipt.tsx

```diff
@@ -173,9 +173,11 @@ export default function TransactionReceipt({ txReceipt, txConfig }: IStepCompone
       <Link to="/dashboard">
         <Button className="TransactionReceipt-back">Back to Dashboard</Button>
       </Link>
-      <Button secondary={true} className="TransactionReceipt-another">
-        Send Another Transaction
-      </Button>
+      <Link to="/send" onClick={() => window.location.reload()}>
```

Comment:
Why the `.reload()` ?! This will reset the entire app...

Reply:
Is there a better way to handle restarting the flow? If you don't add the reload, it doesn't actually route anywhere because you're already at the same location